API/controllers/mealsControllers.js

```diff
@@ -0,0 +1,271 @@
+const {
+  newMealSchema,
+  availableServingsSchema,
+} = require("../validators/validators");
+//GET ALL MEALS
+function getAllMeals(req, res) {
+  let pool = req.pool;
+  let { page, pageSize } = req.query;
+  let offset = (Number(page) - 1) * Number(pageSize);
+  pool.query(
+    `SELECT * FROM menu_items ORDER BY meal_id OFFSET ${offset} ROWS FETCH NEXT ${pageSize} ROWS ONLY`,
+    (err, result) => {
+      if (err) {
+        console.log("error occured in query", err);
+      } else {
+        res.json(result.recordset);
+      }
+    }
+  );
+}
+
+//DELETE MEAL
+function deleteMeal(req, res) {
+  let pool = req.pool;
+  let requestedId = req.params.mealId;
+  pool.query(
+    `DELETE FROM menu_items WHERE meal_id = ${requestedId}`,
+    (err, result) => {
+      //ERROR CHECK
+      if (err) {
+        res.status(500).json({
+          success: false,
+          message: "Internal server error.",
+        });
+        console.log("Error occured in query", err);
+      }
+
+      //CHECK IF REQUESTED USER IS AVAILABLE
+      if (result.rowsAffected[0] === 0) {
```

Comment:
If rows affected is zero why compare to 0, it's already false. Just negate the statement